### ISSUE TITLE
Default values in body-parser (Express.js) implicitly limit batch size

### ISSUE THREAD

**Issue by tiagosimao** [sqlpad/sqlpad#921]
Yey.

Trying to run a batch of queries in the UI one can easily hit the 100k default [payload cap of body-parser](https://github.com/expressjs/), getting a generic 500 error in the client and a  ```PayloadTooLargeError: request entity too large``` in the server logs.

I can PR something, just wondering if one would want to increase the default at https://github.com/sqlpad/sqlpad/blob/fab135364cef68db25416a198e81c3a3bce3e933/server/app.js#L98 or allow for some custom config... guessing the second.

Haven't found references to this problem but doesn't seem very _corner-casy_ for what SQLPad aims to do... would be really glad to help. Any pointers?

**Comment by rickbergfalk** [comment]
How large are you thinking @tiagosimao? I'd be open to making the change without adding config if it is somewhat reasonable (1 MB maybe?).

Using config works too. If you were to go that route it'd be adding an entry to the config-items.js file at https://github.com/sqlpad/sqlpad/blob/master/server/lib/config/config-items.js, accessing the value with `config.get('keyName')` and documenting it in a related config section in docs directory.

**Comment by tiagosimao** [comment]
Hi @rickbergfalk thanks so much for the prompt response.

Having an explicit, documented default (vs side effect coming from upstream) is immediately better I reckon. If that default is opinionated to the context of what SQLPad does, even more so.

I'm just a user that is still just trying out stuff so take my suggestion with a grain of salt
- If people use SQLPad to run queries, a 100k default should be ok even for big weird queries
- If people use SQLPad to also run INSERT/COPY like statements, that cap can be quickly reached

In a specific use-case I have, I'm leveraging on SQLPad to offer a specific group of users a way to run a lot of SQL inserts currently a ~600k payload (previously generated, think on DB dump type of scenario).
Using SQLPad is just better than granting these users direct DB access from their workstations... better in UX, circumvents ad-hoc network setup, authentication and authorization issues (using OIDC)

I digress a bit but my point is I don't know if what I'm trying to solve with SQLPad is something that one would expect to work out of the box, without explicit config.

**TL;DR**
From the little context I have with this project I would:
- Set and document some default (can be 1MB as you suggested, matching nginx's default, a popular reverse proxy)
- Provide a way to configure the payload size, adding to the documentation
- Make the server return the correct HTTP code (413) when this specific error occurs, further helping future users by pointing in the correct direction.

Makes sense?

**Comment by rickbergfalk** [comment]
I agree and think these are all good points. I'm even more convinced 1MB default is the way to go if nginx has that as a default as well.. 

Thanks for sharing your use case BTW! Definitely keep the project posted if there's anything that could be improved.

Out of curiosity is this 600k payload via the API or via the UI? I'm curious how the Ace code editor component will handle a document of that size.

**Comment by tiagosimao** [comment]
Working on it with ongoing PR (don't know how to link it here though)

Its via the UI, a good old paste into the text editor with 1076 lines of inserts with ~500 columns. The editor doesn't even stutter :)
